docs/resources/integration_aws_event_bridge.md

```diff
@@ -0,0 +1,50 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "datadog_integration_aws_event_bridge Resource - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Provides a datadogintegrationawseventbridge resource. This can be used to create and manage Event Sources for each Datadog integrated AWS account.
```

Comment:
```suggestion
  Provides a Datadog - Amazon Web Services integration EventBridge resource. This can be used to create and manage Event Sources for each Datadog integrated AWS account.
```

Reply:
For consistency with other AWS resource descriptions